diff --git a/node_modules/react-native-payments/react-native-payments.podspec b/node_modules/react-native-payments/react-native-payments.podspec
index 52e6d00..3ea0025 100644
--- a/node_modules/react-native-payments/react-native-payments.podspec
+++ b/node_modules/react-native-payments/react-native-payments.podspec
@@ -15,6 +15,6 @@ Pod::Spec.new do |s|
   s.requires_arc     = true
 
   s.dependency 'React'
-  s.dependency 'Stripe', '~> 19'
-  s.dependency 'Braintree', '~> 4'
+  # s.dependency 'Stripe', '~> 19'
+  # s.dependency 'Braintree', '~> 4'
 end
